tool_path_planner/CMakeLists.txt

```diff
@@ -13,6 +13,7 @@ find_package(catkin REQUIRED COMPONENTS
   geometry_msgs
   noether_msgs
   rosconsole
+  roscpp
```

Comment:
We don't need roscpp except for the test_segment_axes.cpp file, which I have thoughts on below.

Reply:
Addressed in 3688fa4